openerp/tools/osutil.py

```diff
@@ -81,7 +81,7 @@ def zip_dir(path, stream, include_dir=True):      # TODO add ignore list
 
     with zipfile.ZipFile(stream, 'w', compression=zipfile.ZIP_DEFLATED, allowZip64=True) as zipf:
         for dirpath, dirnames, filenames in os.walk(path):
-            for fname in filenames:
+            for fname in sorted(filenames):
```

Comment:
@dja-odoo Can you add a comment that explains why this hack? This will be easier to understand for next maintainers of this code